src/ethereum_clis/types.py

```diff
@@ -93,3 +101,19 @@ class TransitionToolOutput(CamelModel):
     alloc: Alloc
     result: Result
     body: Bytes | None = None
+
+
+class TransitionToolState(CamelModel):
+    """Transition tool state."""
```

Comment:
@winsvega how about?
```suggestion
class TransitionToolConfig(CamelModel):
    """Transition tool configuration."""
```

Reply:
Changed to TransitionToolContext in latest commit.